zulipterminal/model.py

```diff
@@ -281,9 +281,11 @@ def _update_initial_data(self) -> None:
                     apply_markdown=True,
                 )
             except zulip.ZulipError as e:
+                self.controller.loading = False
```

Comment:
It would be simpler to handle this in the `Controller` rather than the `Model`, after we've raised an exception; feel free to chat if you want to discuss this.